src/viadot/sources/vid_club.py

```diff
@@ -324,7 +324,6 @@ def get_response(
 
     def to_df(
         self,
-        if_empty: str = "warn",
```

Comment:
Why remove this? This should be used together with the `_handle_if_empty()` `Source` method, same as in other sources, instead of reinventing the wheel. See eg. https://github.com/dyvenia/viadot/blob/2.0/src/viadot/sources/bigquery.py#L243-L244.